docs/snippets/traefik-proxy/traefik-public-private-config.yaml

```diff
@@ -0,0 +1,16 @@
+---
+type: public
+providers:
+  kubernetesCRD:
+    labelSelector: traffic-type in (public, middleware-public, ...other selectors..)
```

Comment:
AFAIK, with ingressClass 
> If the parameter is set, only resources containing an annotation with the same value are processed.
([source](https://doc.traefik.io/traefik/providers/kubernetes-crd/#ingressclass))

If you use ingressClass, why are you also using labelSelector ?

Reply:
Ah ok. So yeah, I changed. My understanding, that I use to have multi-tenant environment, and tenants use to create they own middleware and etc. Maybe this is why

Reply:
Changed